ast/spec.md

```diff
@@ -166,6 +168,18 @@ interface Identifier <: Expression, Pattern {
 
 An identifier. Note that an identifier may be an expression or a destructuring pattern.
 
+
+# PrivateName
+
+```js
+interface PrivateName <: Expression, Pattern {
+  type: "PrivateName";
+  name: string;
```

Comment:
Still feeling like this should be `Identifier`, otherwise we're diverging from the semantics that are used throughout the rest of the AST, where literal text blocks without quotes are `Identifier` nodes.

I agree it's not as pretty nesting-wise, but it seems more consistent.